lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -599,14 +600,24 @@ export class ResponseHandler {
                 serverTokenResponse?.spa_accountid;
         }
 
+        let accountInfo: AccountInfo | null = null;
+        if (cacheRecord.account && idTokenClaims) {
```

Comment:
What about if cacheRecord.account is truthy but idTokenClaims is not? Looks like this will return null for accountInfo

Reply:
Removing the claims check and letting the spread operator deal with ignoring if undefined.